orbitx/flight_gui.py

```diff
@@ -125,6 +90,260 @@ def callback(*_):
         self.recentre_camera(DEFAULT_CENTRE)
     # end of __self__
 
+    ####################### pritvate methods to initial settings ############################
```

Comment:
"pritvate" is mispelled, also is there a way you can format this comment so that it doesn't trigger a pep8 error?